nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -57,15 +57,19 @@ define([
     const CELL_TYPE = 'app-bulk-import';
 
     const div = html.tag('div'),
+        p = html.tag('p'),
         cssCellType = 'kb-bulk-import';
 
     function DefaultWidget() {
         function make() {
             function start() {
                 alert('starting default widget');
+                return Promise.resolve();
```

Comment:
There's still a couple default widgets that live under tabs, this was being a problem.